docs/fsharp/language-reference/fsharp-interactive-options.md

```diff
@@ -36,36 +36,36 @@ Where lists appear in F# Interactive option arguments, list elements are separat
 |Option|Description|
 |------|-----------|
 |**--**|Used to instruct F# Interactive to treat remaining arguments as command line arguments to the F# program or script, which you can access in code by using the list **fsi.CommandLineArgs**.|
-|**--checked**[**+**&#124;**-**]|Same as the **fsc.exe** compiler option. For more information, see [Compiler Options](../../language-reference/compiler-options.md).|
-|**--codepage:&lt;int&gt;**|Same as the **fsc.exe** compiler option. For more information, see [Compiler Options](../../language-reference/compiler-options.md).|
+|**--checked**[**+**&#124;**-**]|Same as the **fsc.exe** compiler option. For more information, see [Compiler Options](language-reference/compiler-options.md).|
```

Comment:
since you're already are on the language-reference folder, it should only be compiler-options.md